Packages/com.unity.inputsystem/InputSystem/Plugins/Android/AndroidGameController.cs

```diff
@@ -36,7 +36,9 @@ public unsafe struct AndroidGameControllerState : IInputStateTypeInfo
         [InputControl(name = "leftTrigger", offset = (uint)AndroidAxis.Ltrigger * sizeof(float) + kAxisOffset)]
         [InputControl(name = "rightTrigger", offset = (uint)AndroidAxis.Rtrigger * sizeof(float) + kAxisOffset)]
         [InputControl(name = "leftStick")]
-        [InputControl(name = "rightStick", offset = (uint)AndroidAxis.Z * sizeof(float) + kAxisOffset)]
+        [InputControl(name = "rightStick")]
+        [InputControl(name = "rightStick/x", offset = (uint)AndroidAxis.Z * sizeof(float))]
+        [InputControl(name = "rightStick/y", offset = (uint)AndroidAxis.Rz * sizeof(float))]
```

Comment:
Please add ////FIXME or ////REVIEW making a note of the non-contiguous state problem here.

Reply:
Ah also, please update the tests. They use the wrong data and this change should actually break the tests.